### ISSUE TITLE
Booster penalties do not apply to SAAR

### ISSUE THREAD

**Issue by Folles** [pyfa-org/Pyfa#1191]
## Bug Report
When setting booster penalty armorrepairamount to yes, SAAR doesn't get affected.

### Expected behavior:
Repaired amor HP/s should show the negative effect coming from the booster penalty.

### Actual behavior:
It doesn't show the negative effect of the booster penalty on armor repair amount HP/s

### Detailed steps to reproduce:
Set negative effect of booster armor repair amount to YES

### Fits involved in EFT format (Edit > To Clipboard > EFT):
any

### Release or development git branch? Please note the release version or commit hash:
1.29.2

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Win 7

**Comment by blitzmann** [comment]
Hey there @Folles 

Thanks for the report. I took a look at a booster that uses this effect (Mindflood in my example), and you're correct. However, after taking a look at the data from CCP, it seems this is intended (or possibly a bug on CCPs side). 

Effect `boosterArmorRepairAmountPenalty`:
```
"modifierInfo": "
	- domain: shipID\n  
	func: LocationGroupModifier\n  
	groupID: 62\n  
	modifiedAttributeID: 84\n  
	modifyingAttributeID: 1142\n  
	operator: 6\n", 
```

Group ID 62 is "Armor Repair Unit". Small ancil armor repper is under group ID 1199 (Ancillary Armor Repairer), so isn't affected by this effect.

I'll verify with @ccplarrikin about the mechanics of this effect and report back.

**Comment by blitzmann** [comment]
@Folles CCP Larrikin agrees it seems to be a defect in EVE. I'll be submitting an official bug report to CCP today detailing it. I'm gonna leave this issue open for now so that I can remember to periodically check on this. :)

**Comment by blitzmann** [comment]
As an update, logged a bug with CCP. Seems to be affecting shield boosts as well (crash booster) (which apparently isn't implemented in pyfa anyway, lol). Will continue to update once the bug report is resolved :)

**Comment by Folles** [comment]
Wow, from such a little thing, you dig up a hole! *hat's off to you*

**Comment by blitzmann** [comment]
Oh, good catch. I'll add that to the bug report - it's also probably a true defect in EVE.

When CCP added ancil reppers, there were a bunch of effects that they forgot to add these to. most were fixed, but I'm not surprised boosters was one of the things overlooked. :)

**Comment by Xepherion** [comment]
Id say none cared about not having debuffs

**Comment by blitzmann** [comment]
> Id say none cared about not having debuffs

>_> I'll take the blame lol

**Comment by DarkFenX** [comment]
Fixed in 647bdb78dfbb53168ce4f87803aaf51615bce84d
